Localization.EntityFramework/project.json

```diff
@@ -0,0 +1,45 @@
+﻿{
+    "webroot": "wwwroot",
+    "version": "1.0.0-*",
+
+    "dependencies": {
+        "EntityFramework.Core": "7.0.0-*",
+        "EntityFramework.InMemory": "7.0.0-*",
+        "Microsoft.AspNet.IISPlatformHandler": "1.0.0-*",
+        "Microsoft.AspNet.Localization": "1.0.0-*",
+        "Microsoft.Extensions.Localization": "1.0.0-*",
+        "Microsoft.AspNet.Server.Kestrel": "1.0.0-*"
+    },
+
+    "commands": {
+        "web": "Localization.EntityFramework"
+    },
+    
+    "compilationOptions": {
+        "warningsAsErrors": true,
+        "emitEntryPoint": true
+    },
+
+    "frameworks": {
+        "dnx451": { },
+        "dnxcore50": {
+            "dependencies": {
+                "System.Globalization": "4.0.11-beta-*",
+                "System.Globalization.Extensions": "4.0.1-beta-*"
+            }
+        }
+    },
+
+    "publishExclude": [
```

Comment:
Just the unneeded publishExclude/exlude items. This project doesn't use node, etc. so there's nothing there to exclude.